src/material_library.cpp

```diff
@@ -301,20 +303,18 @@ void pyne::MaterialLibrary::write_hdf5_nucpath(hid_t db,
                                                std::string nucpath) const {
   int nuc_size;
   nuc_size = nuclist.size();
-  int nuc_data[nuc_size];
-  {
-    int n = 0;
-    for (auto nuc : nuclist) {
-      nuc_data[n] = nuc;
-      n++;
-    }
+  std::vector<int> nuc_data;
+  for (auto nuc : nuclist) {
+    nuc_data.push_back(nuc);
```

Comment:
Maybe do as above (line 290)
```suggestion
  nuc_data.assign(nuclist.begin(), nuclist.end())
```